packages/sdks/javascript/Makefile

```
@@ -0,0 +1,7 @@
+test:
+	@docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .
+	@docker run devopness_sdk-javascript npm run test
+
+build:
+	@docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .
+	@docker run -v sdk-javascript:/usr/app/ devopness_sdk-javascript npm run build-api-models
```

1. It's a bad and risky practice to remove containers while a new version of the image is been built, please don't do that as that could stop running containers and cause undesired side effects. Don't use `build --force-rm` unless really needed, which is not the case here.

2. Referencing `IMAGE` and `TAG` variables simplify code maintenance

3. This folder `sdk-javascript` does not exist. I'd suggest mapping the volume to current working directory using `PWD` shell function

4. As we don't need to keep this container running once the build is completed, the `--rm` flag can be added to `docker run` command, optimizing the usage of resources

```suggestion
	@docker build -f Dockerfile-dev -t $(IMAGE):$(TAG) .
	@docker run devopness_sdk-javascript npm run test

build:
	@docker build -f Dockerfile-dev -t $(IMAGE):$(TAG) .
	@docker run --rm -v $(PWD):/usr/app/ devopness_sdk-javascript npm run build-api-models
```

---

@jfoliveira resolved